### ISSUE TITLE
Tube-like mesh from line mesh node?

### ISSUE THREAD

**Issue by Gerstmann-Bradley** [JacquesLucke/animation_nodes#1629]
If my memory was correct, Line mesh node was improved by @3DSinghVFX during the development of 2.1.
I don't have an actual reference, but I think this node used to have kind of thickness option.
Or it outputs vertices and edge indices by default that allow users to directly attach an "edge to tube" node behind.
Either way, my goal is to generate a "tube-like mesh".

The most simple way to achieve my goal is to **add "vertices" and "edge indices" output by default** to simplify the workflow of edge to tube.
There are other options, but I think it may be too wordy to discuss.

**Comment by OmarEmaraDev** [comment]
I don't recall Kuldeep working on something like this. It seems like you just need a cylinder, why not just use the Cylinder node and rotate it to alight with the direction you want.
If you want a complex tube structure, then you should use Spline From Edges -> Spline Per Branch -> Mesh From Spline.

**Comment by Gerstmann-Bradley** [comment]
> I don't recall Kuldeep working on something like this. It seems like you just need a cylinder, why not just use the Cylinder node and rotate it to alight with the direction you want.
> If you want a complex tube structure, then you should use Spline From Edges -> Spline Per Branch -> Mesh From Spline.

Yes, I was talking about your second point. Here is an example of the issues. I simply think it's kind of awkward to add an additional "mesh info" node for this kind of purpose, especially I thought the old version output 1. mesh, 2. vertices and 3. edge indicies directly to avoid these issues.
 I can attach the pull request, but I don't think it's truly relative to the proposal.
![Edge to tube](https://user-images.githubusercontent.com/55994236/104509499-c5275f00-55b7-11eb-9bf8-f1117b06cad4.png)

**Comment by OmarEmaraDev** [comment]
Okay, I guess we can add separate outputs directly in the node.

**Comment by Gerstmann-Bradley** [comment]
Thank you both for this :) @OmarEmaraDev @3DSinghVFX
